6.2.2000run.py

```diff
@@ -0,0 +1,13 @@
+
+import time
+
+def timer(func, *args, ** kargs):
+     tic = time.perf_counter()
+     func(*args, **kargs)
+     toc = time.perf_counter()
```

Comment:
Alltough I liked the name, rename 'tic' and 'toc'
Variable names - https://workat.tech/machine-coding/tutorial/writing-meaningful-variable-names-clean-code-za4m83tiesy0 - variable names, abbreviation reference is missing